contributors/devel/cherry-picks.md

```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
+   ```bash
+   FORK_REMOTE=<remote-of-your-fork> \
+   UPSTREAM_REMOTE=<remote-of-upstream> \
+   GITHUB_USER=<your-github-user> \
+   ./hack/cherry_pick_pull.sh <remote-of-upstream>/<release-x.xx> <master-branch-pr-number>
+   ```
+   * You will need to run the cherrypick script separately for each patch release you want to cherrypick to.
+
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
+   This should be the most current milestone.
+   * The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager) for the target release branch is responsible for
+   applying the `cherrypick-approved` label.
+   You may cc them with `<@githubusername>` on your cherrypick PR.
 
-## Cherry Pick Review
+## Cherrypick Review
 
-Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
-requested - this should not be the norm, but it may happen.
+Cherrypick pull requests have an additional requirement compared to normal pull
+requests.
+They must be approved specifically for cherrypick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
 
-## Searching for Cherry Picks
+## Searching for Cherrypicks
 
 See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
 status of PRs labeled as `cherrypick-candidate`.
+
+## Troubleshooting Cherrypicks
+
+Contributors may encounter some of the following difficulties when initiating a cherrypick.
+
+- A cherrypick PR does not apply cleanly against an old release branch.
+In that case, you will need to manually fix conflicts and rebase.
+
+- The cherrypick PR includes code that does not pass CI tests.
+In such a case you will have to fetch the auto-generated branch from your fork, amend the problematic commit and force push to the auto-generated branch.
+Alternatively, you can create a new PR, which is noisier.
+
+- In the case of a rebase or cherrypick specific fix, you can run the cherrypick script again and it will automatically force-push to the autogenerated cherrypick branch.
```

Comment:
@guineveresaenger my bad, I don't think should be included here. :)

What I really meant:
This is can only occur if you created a cherry-pick PR before the parent PR has merged. Then something changed in the parent PR. This change should be reflected in the cherry-pick PR. To do so, you can run the script again, which will pick the changes from the parent PR and force push to the autogenerated branch.

But this is not the ideal way the script should be used, so we should probably not include it here. :)

Reply:
@nikhita @guineveresaenger 
i think it was added because i made a comment somewhere in this discussion.
but please feel free to remove / improve it if it feels like an edge case.

there is also the case where you create the cherry pick PR using the script, but then for instance CI tests do not pass or something about the code in the old branch is different, and while the diff applies, changes only to the cherry pick PR have to be made.

Reply:
yeah those are really special circumstances - I'll take it out.